### PR TITLE
[WIP] cosmetic: update account/login elements

### PR DESCRIPTION
**Pull request by jpoechill**
updates:
• updated login and forgot-pass pages
• updated wording and visual elements on main landing page
• add a cleaner, and more spaced footer
• fixed visual line-breaking issue in nav-bar, and removed unnecessary user-icons

photos attached for reference –
Screen Shot 2019-07-29 at 5 10 01 PM
Screen Shot 2019-07-29 at 5 12 18 PM
screencapture-localhost-8000-2019-07-29-17_08_13

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ htdocs
 htdocs/media/
 htdocs/static/.webassets-cache/
 themes/
+installTop/
```
Comment (smirolo) [inline .gitignore]: missing newline at end of file.

File: djaoapp/templates/_menubar.html [hunk 2]
```diff
@@ -3,8 +3,9 @@
     {% if active_organization %}
       <a href="#" class="nav-link dropdown-toggle" style="padding-top:0;padding-bottom:0;" data-toggle="dropdown" role="button"
         aria-haspopup="true" aria-expanded="false">
-        <span class="menubar-label-title">
-            <span data-trnc data-trnc-len="8">{{active_organization.printable_name}}</span> <span class="caret"></span>
+        <span class="menubar-label-title d-inline-block">
+            <span data-trnc data-trnc-len="8">{{active_organization.printable_name}}</span>
```
Comment (smirolo) [inline djaoapp/templates/_menubar.html]: extra whitespace at end of line.

File: djaoapp/templates/_menubar.html [hunk 3]
```diff
@@ -14,7 +15,8 @@
       <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" role="button"
         aria-haspopup="true" aria-expanded="false">
         <span class="menubar-label">
-          {{request.user.username}} <span class="caret"></span>
+          {{request.user.username}}
```
Comment (smirolo) [inline djaoapp/templates/_menubar.html]: extra whitespace at end of line.

File: djaoapp/templates/accounts/login.html [hunk 4]
```diff
@@ -1,36 +1,44 @@
...
-                    <p>
-{% trans %}Don't have an account?{% endtrans %} <i class="fa fa-long-arrow-right"></i> <a href="{{urls.user.register}}{% if next %}?next={{next}}{% endif %}">{% trans %}Sign up{% endtrans %}</a>
-                    </p>
+    <div class="container">
+        <div class="offset-sm-2 col-sm-8 pb-5 mb-4 pt-5">
+            <div class="page-header">
+                <h1 class="text-center">
+                    {% trans %} Sign In {% endtrans %}
+                </h1>
+
```
Comment (smirolo) [inline djaoapp/templates/accounts/login.html]: extra whitespace on blank line.

**Comment by jpoechill** [comment]
note: there were more changes than needed in this single PR, so I will be hoping to update and break changes down into separate batches.

**Comment by jpoechill** [comment]
closed for PR #136, PR #135

**Pull request closed without merging**